src/about.md

```diff
@@ -207,9 +207,9 @@ as are crates that are pending evaluation.
 [error-blog]: https://brson.github.io/2016/11/30/starting-with-error-chain
 [error-chain]: https://docs.rs/error-chain/
 [Rust Libz Blitz]: https://internals.rust-lang.org/t/rust-libz-blitz/5184
-[cat-science-badge]: https://img.shields.io/badge/science--x.svg?style=social
```

Comment:
Wow excellent work :+1: 
Would it be possible to keep the ".svg" part of url?

In this way this would make a perfect drop in replacement that could still be reverted or enrolled with single sed. Currently rollback in case of problems would be quite painful.